gsport.py

```diff
@@ -122,7 +151,7 @@ def __init__(self, argv):
 class Session:
     def __init__(self, options):
         self.options = options
-        self.cookies = http.cookiejar.MozillaCookieJar(filename='gs_cookies.txt')
+        self.cookies = http.cookiejar.MozillaCookieJar(filename=os.path.join(str(Path.home()), '.gs_cookies.txt'))
```

Comment:
```suggestion
        self.cookies = http.cookiejar.MozillaCookieJar(filename=str(Path(Path.home(), '.gs_cookies.txt'))
```
Just want to point out you can join paths using pathlib. Both will work.